File: Zerojudge-d122.py
# Author : ysh
# 2023/12/11 Mon 14:29:40
while True:
    try:
        a = int(input())
        ans = 0
        while a != 0:
            ans = ans + a // 5
            a = a // 5
        print(ans)
    except:
        break
quit()

File: Zerojudge-d122.cpp
// Author : ysh
// 2023/12/11 Mon 14:31:02
#include<bits/stdc++.h>
using namespace std;
#include<slow>
int main() {
    ios::sync_with_stdio(false);
    cin.tie(0);

    int n;
    while(cin>>n) {
        int ans = 0;
        while(n != 0) {
            ans = ans + (n / 5);
            n = n / 5;
        }
        outl(ans);
    }
    return 0;
}